mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -690,708 +569,8 @@ trait Completions { this: MetalsGlobal =>
       override def isNew: Boolean = true
```

Comment:
Leaving `Type`, `New`, and `None` in `object CompletionPosition` because those CompletionPositions are small enough, and I couldn't see the benefit of splitting them into smaller files.
However, it might be better to split them into smaller files for consistency. 🤔

Reply:
I think it would be better to be consistent here, but that's just my opinion. I am fine either way :)